crates/astria-conductor/src/execution_client.rs

```diff
@@ -13,16 +17,26 @@ use tracing::info;
 
 #[async_trait::async_trait]
 pub(crate) trait ExecutionClient: crate::private::Sealed {
```

Comment:
since this trait is `pub(crate)` you can remove the `Sealed` trait

Reply:
@noot this is already done in the single-entry-point PR